hedy.py

```diff
@@ -2823,7 +2823,9 @@ def transpile(input_string, level, lang="en", skip_faulty=True):
         transpile_result = transpile_inner(input_string, level, lang, populate_source_map=True)
 
     except Exception as original_error:
-        if getenv('ENABLE_SKIP_FAULTY', False) and skip_faulty:
+        hedy_amount_lines = len(input_string.split('\n'))
```

Comment:
If you try to run only one line that contains an error, we wanna show it. But now you try to skip it if there's two lines; one with code and another empty. Maybe `.strip()` would solve the issue.
![image](https://github.com/hedyorg/hedy/assets/20051470/ebec10c1-9fdf-450d-a2fe-358ef408949e)

Reply:
Hi @hasan-sh, Amazing catch and you were right! [Strip Hedy code](https://github.com/hedyorg/hedy/pull/4607/commits/b6c2ddd533278b2e2681ec2244e95ee289ddb7d6) solves this issue.